docs/testplan/platform/pmon-test-plan.md

```diff
@@ -0,0 +1,227 @@
+         * fancontrol : "/usr/share/sonic/platform/fancontrol"
+      - Add the daemon as "skipped_daemons" list if the platform configuration file doesn't exist for a daemon to start.
+
+   d. Verify the expected daemon list from the output of a, b, and c with "/etc/supervisor/conf.d/supervisord.conf"
+      - Check the output of "docker exec pmon bash -c '[ -f /etc/supervisor/conf.d/supervisord.conf ] \
+      && cat /etc/supervisor/conf.d/supervisord.conf | grep program'"
+      - Compare the output from the above steps which is {"all_daemons" - "skipped_daemons"}
+      ```
+   admin@sonic:~$ sudo docker exec pmon bash -c '[ -f /etc/supervisor/conf.d/supervisord.conf ]  && cat /etc/supervisor/conf.d/supervisord.conf | grep program'
+   [program:rsyslogd]
+   [program:lm-sensors]
+   [program:pcied]
+   [program:psud]
+   [program:syseepromd]
+   [program:xcvrd]
+      ```
+   
+   e. Verify each daemon status using "sudo docker exec pmon supervisorctl status"
+      - com
+      - Example) "skip_ledd"==true && "skip_thermalctld"==true && none of three config files in c. available
+      ```
+   admin@sonic:~$ sudo docker exec pmon supervisorctl status
+   supervisor-proc-exit-listener    RUNNING   pid 24, uptime 1:30:24
+   dependent-startup                EXITED    Jan 25 08:23 AM
+   rsyslogd                         RUNNING   pid 27, uptime 1:30:22
+   pcied                            RUNNING   pid 36, uptime 1:30:20
+   psud                             RUNNING   pid 34, uptime 1:30:20
+   syseepromd                       RUNNING   pid 35, uptime 1:30:20
+   xcvrd                            RUNNING   pid 32, uptime 1:30:20
+      ```
+
+3. Perform the following steps for the list of the expected daemon
+	- Verify the daemon is running and report the daemon running status
+
+## Test case # 2 Data for chassisd daemon
+### Test objective
+Verify the expected data for chassisd daemon 
+   - condition to start : "skip_chassisd" != true and [ -e "/usr/share/sonic/platform/chassisdb.conf" ]
+### Test steps
+1. Verify the chassisd running status
+   - If it's not running, report the error status and go to the next 
+2. Verify that module db data setting
+   ```
+   CHASSIS_INFO_TABLE = 'CHASSIS_TABLE'
+   CHASSIS_INFO_KEY_TEMPLATE = 'CHASSIS {}'
+   CHASSIS_INFO_CARD_NUM_FIELD = 'module_num'
+
+   CHASSIS_MODULE_INFO_TABLE = 'CHASSIS_MODULE_TABLE'
+   CHASSIS_MODULE_INFO_KEY_TEMPLATE = 'CHASSIS_MODULE {}'
+   CHASSIS_MODULE_INFO_NAME_FIELD = 'name'
+   CHASSIS_MODULE_INFO_DESC_FIELD = 'desc'
+   CHASSIS_MODULE_INFO_SLOT_FIELD = 'slot'
+   CHASSIS_MODULE_INFO_OPERSTATUS_FIELD = 'oper_status'
+   ```
+
+## Test case # 3 Data for lm-sensors daemon
+### Test objective
+Verify the expected data for lm-sensors daemon
+   - condition to start : "skip_sensors" != true and [ -e "/usr/share/sonic/platform/sensors.conf" ]
+### Test steps
+1. Verify the lm-sensors running status
+   - If it's not running, report the error status and go to the next 
```

Comment:
please check this one: https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/test_platform_info.py#L50